main-actions/src/test/scala/sbt/compiler/EvalTest.scala

```diff
@@ -62,35 +63,38 @@ val p = {
 
   property("val test") = secure {
     val defs = (ValTestContent, 1 to 7) :: Nil
-    val res = eval.evalDefinitions(defs, new EvalImports(Nil, ""), "<defs>", None, "scala.Int" :: Nil)
+    val res =
+      eval.evalDefinitions(defs, new EvalImports(Nil, ""), "<defs>", None, "scala.Int" :: Nil)
     label("Val names", res.valNames) |: (res.valNames.toSet == ValTestNames)
   }
 
   property("explicit import") = forAll(testImport("import math.abs" :: Nil))
   property("wildcard import") = forAll(testImport("import math._" :: Nil))
-  property("comma-separated imports") = forAll(testImport("import annotation._, math._, meta._" :: Nil))
-  property("multiple imports") = forAll(testImport("import annotation._" :: "import math._" :: "import meta._" :: Nil))
+  property("comma-separated imports") = forAll(
+    testImport("import annotation._, math._, meta._" :: Nil))
+  property("multiple imports") = forAll(
+    testImport("import annotation._" :: "import math._" :: "import meta._" :: Nil))
 
-  private[this] def testImport(imports: Seq[String]): Int => Prop = i =>
-    value(eval.eval("abs(" + i + ")", new EvalImports(imports.zipWithIndex, "imp"))) == math.abs(i)
+  private[this] def testImport(imports: Seq[String]): Int => Prop =
+    i =>
+      value(eval.eval("abs(" + i + ")", new EvalImports(imports.zipWithIndex, "imp"))) == math.abs(
+        i)
```

Comment:
This is a known issue https://github.com/scalameta/scalafmt/issues/667 I suspect it would look nicer wrapped in curly braces.